test/ably/restcrypto_test.py

```diff
@@ -67,10 +67,10 @@ def test_crypto_publish_text(self):
         publish0.publish("publish2", 24.234)
         publish0.publish("publish3", six.u("This is a string message payload"))
         publish0.publish("publish4", six.b("This is a byte[] message payload"))
-        publish0.publish("publish5", {"test": "This is a JSONObject message payload"})
-        publish0.publish("publish6", ["This is a JSONArray message payload"])
+        publish0.publish("publish5", '{"test": "This is a JSONObject message payload"}')
```

Comment:
Again, you can't publish JSON in a string and expect this to be a valid test, you are simply testing that the string matches.